src/gt4py/next/otf/compilation/build_systems/cmake.py

```diff
@@ -50,6 +52,8 @@ def __call__(
         cmake_languages = [cmake_lists.Language(name="CXX")]
         if source.program_source.language is languages.Cuda:
             cmake_languages = [*cmake_languages, cmake_lists.Language(name="CUDA")]
+        elif source.program_source.language is languages.Hip:
+            cmake_languages = [*cmake_languages, cmake_lists.Language(name="HIP")]
```

Comment:
Optional suggestion. The current code is ugly and boring but I'm not sure if this suggestion actually improves anything:
```suggestion
        if (src_lang:=prototype_program_source.language) in [languages.Cuda, languages.Hip]: 
            cmake_languages = [*cmake_languages, cmake_lists.Language(name=src_lang.__name__.upper())]
```

Also, out of the scope of this PR, but it's a bit weird that this piece code is replicated exactly in another file...